nicolegooden/ideabox-boilerplate
# Rename input field section to form for semanatic purposes

What’s this PR do?
- `index.html`: Changed input section form and changed the button type to be disabled on page load
- `styles.css`: Add overflow idea for side panel bug
- `main.js`: Created `disableSaveButton(event)` which disables save button when input field are empty and enables it when data is entered into both fields

  Where should the reviewer start?  
- Chrome browser

How should this be manually tested?  
- Open in chrome browser and enter data into each text field to make sure the save button enables.

Any background context you want to provide?  
- I received some tutoring from John Adams for the `disableSaveButton(event)`

What are the relevant tickets?  
- N/A

Screenshots (if appropriate)
- N/A

  Questions: 
- We may need to change the querySelectorsAll for the input values.
